velox/dwio/common/IntDecoder.cpp

```diff
@@ -164,6 +188,40 @@ void IntDecoder<isSigned>::bulkReadRowsFixed(
   }
 }
 
+template <bool isSigned>
+template <typename T>
+T IntDecoder<isSigned>::readInt() {
```

Comment:
Can we keep these 2 functions in header so that they can be inlined if needed?

Reply:
OK, move these to the header file